### PR TITLE
Add ProFind.app v1.7.0

### PR DESCRIPTION
**Pull request by tnira**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].
- [x] Attempted to find an [appcast].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256
[appcast]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md

### CODE REVIEW

File: Casks/profind.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'profind' do
+  version '1.7.0'
+  sha256 '21391307ebfa022a90ff9fcd8706d88ef56999e5b5f3e68868e46e514da8fbab'
+
+  url "https://www.zeroonetwenty.com/profind/downloads/ProFind#{version.no_dots}.dmg"
+  name ''
```
Comment (vitorgalvao) [inline Casks/profind.rb]: You forgot this.

File: Casks/profind.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+cask 'profind' do
+  version '1.7.0'
+  sha256 '21391307ebfa022a90ff9fcd8706d88ef56999e5b5f3e68868e46e514da8fbab'
+
+  url "https://www.zeroonetwenty.com/profind/downloads/ProFind#{version.no_dots}.dmg"
```
Comment (vitorgalvao) [inline Casks/profind.rb]: Please add an [`appcast`](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

`https://www.zeroonetwenty.com/profind/downloads/ProFindVersionInfo.xml`

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/profind.rb (hunk 2 above)
Comment (tnira) [reply Casks/profind.rb]: I fixed it.

**Comment by vitorgalvao** [comment]
Thank you.

**Pull request merged**